chaosazure/common/resources/graph.py

```diff
@@ -29,15 +30,17 @@ def fetch_resources(input_query: str, resource_type: str,
         raise InterruptExecution(msg)
 
     # prepare results
-    results = __to_dicts(resources.data, client.api_version)
+    results = __to_dicts(resources.data, "2021-03-01")
```

Comment:
What's the impact in future release of pinning this?

Reply:
The **ResourceGraphClient** class doesnot have an **api_version** attribute, the api version is an attribute in the protected member attribute _config, you have to access it like client._config.api_version. Suggest what should be done.

I suggest we remove the version variable from __to_dicts function signature, and stop doing a if else inside, support only >= 2019-04-01 version, since we are going to pin only new versions in the requirements.